doc/source/api/api.rst

```diff
@@ -200,5 +200,5 @@ This Command Line Interface (CLI) tool is used to manage Covalent server.
 
 .. click:: covalent_dispatcher._cli.cli:cli
     :prog: covalent
-    :commands: start,stop,restart,status,purge,logs
+    :commands: start,stop,restart,status,purge,logs,migrate_legacy_result_object
```

Comment:
@cjao for some reason I dont see it here https://covalent--976.org.readthedocs.build/en/976/api/api.html#covalent-cli-tool

Reply:
I don't see it either. Thanks for the catch!

Reply:
Forgot that `click` converts underscores to hyphens.